src/politech_backend/urls.py

```diff
@@ -44,3 +45,4 @@
 ]
 
 urlpatterns = format_suffix_patterns(urlpatterns)
+urlpatterns += staticfiles_urlpatterns() # Needed to load static files for Swagger
```

Comment:
Nitpicking: Not a fan of comments within the code.

IMO it belongs as a comment here on Github.
If a developer then feels the need to remove the line, they can look it up via Github